pkgs/development/python-modules/aiosmtplib/default.nix

```diff
@@ -36,6 +36,10 @@ buildPythonPackage rec {
     trustme
   ];
 
+  disabledTests = [
+    "test_connect_via_socket_path"
```

Comment:
Please add a comment about the issue with this test.

Works on [Hydra](https://hydra.nixos.org/eval/1788526?filter=aiosmtplib&compare=1788514&full=#tabs-still-succeed) and locally for me.